src/Utilities/DeclarationBlockParser.php

```diff
@@ -90,12 +90,7 @@ public function parse(string $declarationBlock): array
                 continue;
             }
 
-            \assert(\is_array($matches));
             $propertyName = $matches[1];
-            if ($propertyName === '') {
-                // This cannot happen since the regular expression matches one or more characters.
-                throw new \UnexpectedValueException('An empty property name was encountered.', 1727046409);
-            }
```

Comment:
Has it really delved into the regex to determine this?  If so, that's impressive.